### DISCUSSION TITLE
when creating profile, wrong list of ssh keys are in ssh key dropdown

### DISCUSSION THREAD

**Discussion by alsoeric** [borgbase/vorta#1421] [FAQ] [answered]
I created 2 new keys for borg backups. put them into .ssh, started vorta, by new keys were not listed. I cleaned out about of old keys, new keys were not listed and old, removed keys were still there.  is there a way to refresh the list of keys to only the current ones?

**Comment by real-yfprojects** [comment, marked as answer]
Quitting and restarting vorta should refresh the list.

**Reply by alsoeric** [reply]
I thought that should be the case as well but I restarted vorta a couple of times in the list didn't change. I had lunch, came back in the list was as it should be. I'll play around with a bit more and see if I can get a reproducible case.
